TAF_EasyRest/TestFramework/PageComponents/NavigationMenuPageComponent.cs

```diff
@@ -0,0 +1,66 @@
+﻿
+namespace TestFramework.Pages
```

Comment:
maybe we should create another namespace for components?

Reply:
We'll discuss...